checkov/terraform/checks/resource/gcp/GKEUseCosImage.py

```diff
@@ -12,24 +12,36 @@ def __init__(self):
 
     def scan_resource_conf(self, conf):
         """
-            Looks for password configuration at azure_instance:
-            https://www.terraform.io/docs/providers/google/r/compute_ssl_policy.html
-        :param conf: google_compute_ssl_policy configuration
-        :return: <CheckResult>
+        CKV_GCP_22
+        error creating NodePool: googleapi: Error 400: Creation of node pools using node images based on Docker
```

Comment:
I did not understand this documentation change. Seems like the previous one was better and you might want to add on it, not delete it.